man/systemd.socket.xml

```diff
@@ -130,7 +121,9 @@
   </refsect1>
 
   <refsect1>
-    <title>Automatic Dependencies</title>
+    <title>Implicit Dependencies</title>
+
+    <para>The following dependencies are implicitly added.</para>
```

Comment:
maybe end this in a colon, and then use an itemizedlist for the paragaphs? (here and everywhere else)

Reply:
Okay, I will change it. Thanks.